data_graph/views.py

```diff
@@ -0,0 +1,34 @@
+'''Provide API views for data-graph module.'''
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+
+from data_graph.services import DataGraphService, DataGraphParamsService
+
+
+class DataGraphView(APIView):
```

Comment:
You might want to use `viewsets.Viewset`, since we prepare to add more actions on the view, it's boring to register them one at a time, by using `viewsets.Viewset`, you can register all actions along with custom actions quickly.